.github/workflows/build.yml

```diff
@@ -81,7 +81,13 @@ jobs:
       run: nix develop --command bash -c "sudo make server-run" &
 
     - name: Run filesystem
-      run: nix develop --command bash -c "sudo ./filesystem/filesystem -f -h=127.0.0.1 mount-dir" &
+      run: nix develop --command bash -c "sudo ./filesystem/filesystem -h=127.0.0.1 mount-dir"
+
+    - name: Check filesystem is mounted
+      run: findmnt mount-dir
+
+    - name: Run filesystem
```

Comment:
Maybe let's name it "Run 2nd filesystem" for clarity

Reply:
Good point